common/jsonrpc_errors.h

```diff
@@ -64,4 +64,6 @@ static const errcode_t INVOICE_WAIT_TIMED_OUT = 904;
 /* Errors from HSM crypto operations. */
 static const errcode_t HSM_ECDH_FAILED = 800;
 
+static const errcode_t PLUGIN_PENDING_REQUEST_DIED = 1000;
```

Comment:
I don't think we can make this a normal error; I think PLUGIN_ERROR or maybe a new PLUGIN_DIED (with associated negative code) is more appropriate?